test/run.js

```diff
@@ -1,44 +1,48 @@
 /* Copyright 2020 Record Replay Inc. */
 
 // Harness for end-to-end tests. Run this from the devtools root directory.
-
 const fs = require("fs");
 const http = require("http");
 const os = require("os");
 const { spawnSync, spawn } = require("child_process");
 const { defer } = require("../src/protocol/utils");
+const url = require("url");
```

Comment:
We should update  `main.js` so that run is called with `run --record-all`
https://github.com/RecordReplay/devtools/blob/master/.github/actions/test/main.js#L34